sdcm/coredump.py

```diff
@@ -301,11 +301,12 @@ def get_list_of_cores(self) -> Optional[List[CoreDumpInfo]]:
         #     Tue 2020-01-14 16:18:56 +07    6321  1000  1000   3 present   /usr/bin/scylla
         #     Tue 2020-01-14 16:31:39 +07   18554  1000  1000   3 present   /usr/bin/scylla
         for line in result.splitlines():
-            if "bash" in line:
+            if "bash" in line or "fwupd" in line:
```

Comment:
`"bash" in line` not sure is a good idea, as there might be in a real coredump as well